src/dialog/README.md

```diff
@@ -0,0 +1,111 @@
+## Introduction
+
+The Material Design Lite (MDL) **dialog** component allows for verification of
+user actions, simple data input, and alerts to provide extra information to users.
+
+## Basic Usage
+
+To use the dialog component, you must be using a browser that supports the [dialog element](http://www.w3.org/TR/2013/CR-html5-20130806/interactive-elements.html#the-dialog-element).
+Only Chrome and Opera have native support at the time of writing.
+For other browsers you will need to include the [dialog polyfill](https://github.com/GoogleChrome/dialog-polyfill) or create your own.
+
+Once you have dialog support create a dialog element.
+Within that container, add a content element with the class `mdl-dialog__content`.
+Add you content, then create an action container with the class `mdl-dialog__actions`.
+Finally for the markup, add your buttons within this container for triggering dialog functions.
+Keep in mind, the order is automatically reversed for actions.
```

Comment:
It stems from these excerpts of the specification:

> Affirmative actions are placed on the right side and continue the process. Affirmative actions may be destructive, like “Delete” or “Remove.”

> Dismissive actions are placed directly to the left of affirmative actions and return the user to the originating screen or step in the process.

Typically, developers write their affirmative parts *first* when coding then do the dismissive stuff. This way it just helps developers not need to think about their writing order to be spec compliant. Perhaps I should add an inline note about this so developers understand why it reverses by default.